src/components/MovieList.tsx

```diff
@@ -17,7 +18,15 @@ export default function MovieList(props: {
       My movies:
       {props.movies.map((movie, index) => (
         <div className="movieOption" key={index}>
-          {movie.title}
+          <div className="movieOptionWrapper">
```

Comment:
OOC why can't this all be in the main movieOption div? It seems like a bit more to keep track of

Reply:
styling with flexbox to get the emoji and title on the left, and the x on the right